### ISSUE TITLE
LuLu keeps asking for Xcode sim apps...

### ISSUE THREAD

**Issue by skrew** [objective-see/LuLu#341]
... because path always changes between builds on the simulator.

After a few research, it's look like an option was available but removed since 2.0.

Any news / tip about that ?

**Comment by hoanngothanh** [comment]
Yes. I also face this issue. It is annoying that Lulu keep asking permission every time Xcode build

**Comment by hamza02x** [comment]
It's so annoying.

**Comment by dmelgar** [comment]
Same here. This is so annoying it makes it necessary to disable LuLu while I'm developing.
